src/components/SearchForm/SearchForm.js

```diff
@@ -0,0 +1,32 @@
+import "./SearchForm.css";
+import React from "react";
+
+function SearchForm({ onSubmit }) {
+  const [input, setInput] = React.useState("");
+
+  function handleInputChange(e) {
+    setInput(e.target.value);
+  }
+
+  function handleSubmit(e) {
+    e.preventDefault();
+    onSubmit(input);
+  }
+
+  return (
+    <form className="search-form" name="search-form" onSubmit={handleSubmit}>
+      <input
+        className="search-form__input"
+        type="text"
+        placeholder="Enter topic"
+        onChange={handleInputChange}
```

Comment:
If you have `onChange` or ` onInput` in an input then you need to add attribute `value` to this input.
This is how React controls inputs.
To read more about **controlled components**: https://reactjs.org/docs/forms.html#controlled-components